Add Tweet cards

Bravemobin opened the pull request:
I've added the Tweet section, changed the page.tsx and changed next.config.ts for images.

the tweets.ts in the data folder needs to changed to use real data

AmirHosseinKarimi committed:
build: add packageManager to package.json

AmirHosseinKarimi committed:
Merge pull request #1 from frontChapter/test

Fix lint test

Bravemobin committed:
feat: Add tweet marquee component

Bravemobin committed:
feat: Add heading to tweet section

Bravemobin committed:
feat: Add tweetsData file for dynamic tweets

Bravemobin committed:
feat: Add image optimization for avatars

fulcain committed:
refactor: change all faq related files and paths naming conventions

fulcain committed:
refactor: change the imported data filename into the FAQ component

This is the previous commit's leftover

fulcain committed:
refactor: move the FAQ section under app/components

fulcain committed:
build: run pnpm install

fulcain committed:
feat: Add FaqDataType to the faqData array

fulcain committed:
refactor: change the faq data type naming to match other files and folders

Bravemobin committed:
fix: Correct import path for TweetSection component

fulcain committed:
feat: add dana font to project

fulcain committed:
feat: add danaVF font to html using localFont method

fulcain committed:
feat: add background and color to body

fulcain committed:
feat: add font callback for danaVF

fulcain committed:
refactor: change favicon

fulcain committed:
refactor: change app title

fulcain committed:
refactor: change the faq question data

fulcain committed:
feat: add background color to body

fulcain committed:
style: refactor the file

fulcain committed:
feat: add 12 spacing according to UI

fulcain committed:
refactor: change the gap of FAQ section from gap-8 to gap-12

fulcain committed:
feat: add 20 and 24 spacing to tailwind config

fulcain committed:
feat: add 8 spacing to tailwind

fulcain committed:
refactor: change the section component spacing to match design

fulcain committed:
refactor: change the spacing for mobile and desktop for section component

AmirHosseinKarimi commented:
خسته نباشی مبین جان
لطفا تم پیشفرض رو روی دارک بذار
<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Firefox 134.0 on Linux x86_64
Viewport:  2560 x 1295 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-9--fc-nextjs.netlify.app%2F&os=OS%20X&os_version=Big%20Sur&browser=Firefox&browser_version=134.0&resolution=2560x1295&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-9--fc-nextjs.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=679e9467074e7a00084cb1c1&commentID=67a8712b2d93a676c3b5c165&resolution=resolved)

AmirHosseinKarimi commented:
![image](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiYWFmODkwZDMtODMzOS00MTVlLTkxNTgtOGQ3NjE2MDUxZjE0IiwiZGVwbG95X2lkIjoiNjc5ZTk0NjcwNzRlN2EwMDA4NGNiMWMxIiwiaWQiOiJlN2Y2YTc0Mi0zYmE1LTQ4MjktOGY3Yi1jMDk5YWU5ZTlkMzIifQ==)

صفحه و کامپوننت هم بصورت تمام‌عرض (Full-width) باشه
<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Firefox 134.0 on Linux x86_64
Viewport:  2560 x 1295 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-9--fc-nextjs.netlify.app%2F&os=OS%20X&os_version=Big%20Sur&browser=Firefox&browser_version=134.0&resolution=2560x1295&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-9--fc-nextjs.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=679e9467074e7a00084cb1c1&commentID=67a87150bbb97dfeb4907806&resolution=resolved)

AmirHosseinKarimi commented:
اسم و تصاویر و محتوای اولیه رو هم بر اساس محتوای داخل دیزاین قرار بده
<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Firefox 134.0 on Linux x86_64
Viewport:  2560 x 1295 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-9--fc-nextjs.netlify.app%2F&os=OS%20X&os_version=Big%20Sur&browser=Firefox&browser_version=134.0&resolution=2560x1295&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-9--fc-nextjs.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=679e9467074e7a00084cb1c1&commentID=67a87182a066fb0feb8ba39e&resolution=resolved)

AmirHosseinKarimi reviewed (changes requested):
مبین جان لطفا موارد ذکر شده رو اعمال کن.

fulcain committed:
refactor: change some faq component styles

I've changed font related styles

fulcain committed:
refactor: change localFont method with direct use of font face

fulcain committed:
refactor: change the way font was implemented to localFonts

After doing more research I found out the best way is using the
localFont and not use the css file directly

fulcain committed:
refactor: remove the styles related to body applied in css

fulcain committed:
refactor: move the type file related to faq next to FAQ component

fulcain committed:
refactor: change the import paths to use alias path

fulcain committed:
refactor: remove the id prop from faq data and use index instead

fulcain committed:
fix: change .variable to .className for localFont

fulcain committed:
refactor: remove utility classes in global css

fulcain committed:
refactor: remove all css variables and apply them directly in tailwind

fulcain committed:
feat: add dark styles for components

fulcain committed:
refactor: remove id prop from faq data file

Bravemobin committed:
style: change width of tweet component

Bravemobin committed:
refactor: dark mode and user infos

Bravemobin commented:
همه موارد ذکر شده اعمال شد
فقط عکس هایی که گفتین کانفیگشو گزاشتم که url عکسو داخلش بزارین

AmirHosseinKarimi committed:
feat: update FAQ section data

AmirHosseinKarimi committed:
build: add commit-and-tag-version package

AmirHosseinKarimi committed:
ci: add pump command to pump package version

AmirHosseinKarimi committed:
chore(release): 0.2.0

Co-authored-by: Majid Kargar <majidkargar01@gmail.com>

AmirHosseinKarimi committed:
Merge pull request #7 from frontChapter/faq

Add FAQ section

AmirHosseinKarimi committed:
ci: run changelog workflow only for new pull-requests

AmirHosseinKarimi committed:
ci: setup all-contributor bot

AmirHosseinKarimi committed:
chore(release): 0.2.1

allcontributors[bot] committed:
docs: update README.md

allcontributors[bot] committed:
docs: update .all-contributorsrc

AmirHosseinKarimi committed:
Merge pull request #15 from frontChapter/all-contributors/add-AmirHosseinKarimi

docs: add AmirHosseinKarimi as a contributor for projectManagement, infra, and 3 more

allcontributors[bot] committed:
docs: update README.md

allcontributors[bot] committed:
docs: update README.md

allcontributors[bot] committed:
docs: update .all-contributorsrc

AmirHosseinKarimi committed:
Merge pull request #16 from frontChapter/all-contributors/add-fulcain

docs: add fulcain as a contributor for code

Bravemobin committed:
Merge branch 'dev' of https://github.com/frontChapter/NextJS into tweet-cards

Bravemobin committed:
refactor: Move Marquee and TweetCard components

AmirHosseinKarimi reviewed (changes requested):
Package managers conflict. PNPM used as project package manager but Bun has been used for this PR.

Bravemobin committed:
chore: remove bun.lockb file

AmirHosseinKarimi committed:
Merge branch 'dev' into tweet-cards

AmirHosseinKarimi committed:
fix: revert accidentally removed code
